source/CommonJobs/CommonJobs.MVC.UI/Scripts/Evaluations/dashboard.js

```diff
@@ -243,32 +244,39 @@
                     this.calificationActionTooltip("Evaluar como responsable");
                     this.calificationActionClass("icon user");
                     this.calificationActionText("Evaluar");
+                    this.sortByPriority(0);
                     return;
                 } else if (this.state() == 1 || this.state() == 3) {
                     this.calificationActionText("Evaluar");
                     this.calificationActionClass("icon empresa");
                     this.calificationActionTooltip("Evaluar como empresa");
-                    return;
+                    this.sortByPriority(2);
+                    return
                 } else if (this.state() == 5) {
                     this.calificationActionText("Devolución");
                     this.calificationActionClass("icon user");
                     this.calificationActionTooltip("Hacer la devolución con el evaluado");
+                    this.sortByPriority(3);
                     return;
                 }
             } else if (!this.isResponsible && (this.state() == 4 || this.state() == 5)) {
                 this.calificationActionTooltip("Lista para devolucion");
                 this.calificationActionClass("icon view");
                 this.calificationActionText("Ver Evaluación");
+                this.sortByPriority(4);
                 return;
             } else if (!this.isResponsible && this.isEditable && this.state() != 6) {
                 this.calificationActionTooltip("Evaluar como evaluador");
                 this.calificationActionClass("icon user");
                 this.calificationActionText("Evaluar");
+                this.sortByPriority(1);
                 return;
             }
             this.calificationActionText("Ver Evaluación");
             this.calificationActionClass("icon view");
             this.calificationActionTooltip("Ver Evaluación");
+            this.sortByPriority(5);
```

Comment:
Yes, I understand what you are saying! `If the priority was built exactly for this, then maybe it could be named differently to reflect that.` that the idea...  What name do you suggest?